### PR TITLE
(maint) modsync update - stable

### PR DESCRIPTION
**Pull request by glennsarti**
The PR contains the updates from the new modsync config for windows;

- LICENSE, NOTICE, CONTRIBUTING, MAINTAINERS updates (where applicable)
- Gemfile, Appveyor, Travis, gitignore cleanup

https://github.com/puppetlabs/modulesync_configs/pull/66
https://github.com/puppetlabs/modulesync_configs/pull/63
https://github.com/puppetlabs/modulesync_configs/pull/62
https://github.com/puppetlabs/modulesync_configs/pull/61

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Puppet Module - puppetlabs-registry
+
+Copyright 2012 - 2016 Puppet, Inc.
```
Comment (ferventcoder) [inline NOTICE]: Original copyright was set at 2013. Not sure if there was a reason.

File: appveyor.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 version: 1.1.x.{build}
 skip_commits:
-  message: /(^\(?doc\)?.*|.*[A|a]cceptance [T|t]est.*)/
+  message: /^\(?doc\)?.*/
```
Comment (ferventcoder) [inline appveyor.yml]: 👍

File: appveyor.yml [hunk 3]
```diff
@@ -24,7 +24,6 @@ matrix:
   fast_finish: true
 install:
 - SET PATH=C:\Ruby%RUBY_VER%\bin;%PATH%
-- gem install bundler --quiet --no-ri --no-rdoc
```
Comment (ferventcoder) [inline appveyor.yml]: This was still here? 

Oh, this must be targeted at stable!

File: .travis.yml [hunk 4]
```diff
@@ -16,8 +16,5 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="3.3.0"
-  allow_failures:
-    - rvm: 2.1.5
-      env: PUPPET_GEM_VERSION="~> 4.0"
```
Comment (ferventcoder) [inline .travis.yml]: Well this opened a can of worms. Good though.

**Comment by ferventcoder** [comment]
I think we may want to merge some of the fixes from master down to stable first?

File: appveyor.yml (hunk 3 above)
Comment (glennsarti) [inline appveyor.yml]: Yes stable first. Current thinking is I may not be able to merge this stuff up to master.  Instead, fresh PRs for master

File: NOTICE (hunk 1 above)
Comment (glennsarti) [inline NOTICE]: First ever commit was in 2012

File: .travis.yml (hunk 4 above)
Comment (glennsarti) [inline .travis.yml]: Yes. Yes it did.  I need to modify modysnc settings to try and import this stuff

File: NOTICE (hunk 1 above)
Comment (ferventcoder) [inline NOTICE]: 👍

**Pull request merged**